### PR TITLE
##Hello Microverse 

### PR DESCRIPTION
**Pull request by farouqdaif1**
1. create the repository 
2. make branch 
3. add gitgnore flie 
4. set up linters
5. Add README file 
6. Add 3 files HTML CSS test,md 

thanks for reviewing ❤️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,76 @@
...
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] The readme file should be descriptive, here I have noticed that there is a duplicate of the data on top.

File: README.md [hunk 2]
```diff
@@ -1 +1,76 @@
...
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages:Html and CSS
+- Technologies used :GIT and GitHup 
+
+
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] This area should contain information on how someone can get a copy of the project and run it on his/her machine. Take a look again at [this](https://github.com/microverseinc/readme-template) link to see the steps on getting started.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
## Hi, @farouqdaif1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
**You have done a good job especially on the descriptive PR and adding the necessary files as required**


## Required Changes ♻️
_Check the comments under the review._👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @farouqdaif1  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**